packages/io/tests/__mocks__/MockCodec.ts

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable @typescript-eslint/no-empty-function */
-import { EncodedObject } from '@ash.ts/io/src/enginecodecs/EncodedData';
+import { EncodedObject } from '@ash.ts/io';
```

Comment:
Please change this to `../../src/enginecodecs/EncodedData` or use `@ash.ts/*` everywhere in the tests.